src/api/search.ts

```diff
@@ -86,6 +86,13 @@ const getProductSearch = async ({
     eq: 'true',
   };
 
+  const discontinuedFilter = {
+    attribute: 'pcm_product_salability',
```

Comment:
If this is missing, it filters out the product or shows them ?

Reply:
if this is missed , it seems product is filtered out. 
I believe this would be available in each product.

Reply:
Please validate - as this wasn't the case with other attributes.  In other cases, the implementation was dependent on that particular attribute being present. Here seems that it is the other way around.

Reply:
Hi @rhudea 
I have validated with the customer `pcm_product_salability` will always be the part of the attribute list.
can you please review so that we can close this as ticket is already approved by customer.